### ISSUE TITLE
Inability to share file on nextcloud

### ISSUE THREAD

**Issue by despens** [n8n-io/n8n#3961]
**Describe the bug**
While other functions of the nextcloud node seem to work fine, sharing files produces an error.

**To Reproduce**

1. Create a nextcloud node
2. Pick Operation "Share"
3. Pick a valid file path
4. Pick Share Type "Public Link"
5. Execute the node

**Expected behavior**
The file should be publicly shared

**Environment (please complete the following information):**

- nextcloud 24.0.3
- n8n Version 0.191.1
- Operation mode `own`

**Additional context**
Both n8n and nextcloud are deployed in standard docker containers.

The nextcloud Web DAV URL was set up as `https://{my-nextcloud-instance}/remote.php/webdav`

API error looks like this:

```
ERROR: UNKNOWN ERROR - check the detailed error for more information
<?xml version="1.0" encoding="utf-8"?> <d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns"> <s:exception>Sabre\DAV\Exception\NotImplemented</s:exception> <s:message>There was no plugin in the system that was willing to handle this POST method.</s:message> </d:error> 
```

<details>
<summary>Details</summary>

```json
{
    "context": {},
    "name": "NodeApiError",
    "cause": {
        "message": "501 - \"<?xml version=\\\"1.0\\\" encoding=\\\"utf-8\\\"?>\\n<d:error xmlns:d=\\\"DAV:\\\" xmlns:s=\\\"http://sabredav.org/ns\\\">\\n  <s:exception>Sabre\\\\DAV\\\\Exception\\\\NotImplemented</s:exception>\\n  <s:message>There was no plugin in the system that was willing to handle this POST method.</s:message>\\n</d:error>\\n\"",
        "name": "Error",
        "stack": "Error: Request failed with status code 501\n    at createError (/usr/local/lib/node_modules/n8n/node_modules/axios/lib/core/createError.js:16:15)\n    at settle (/usr/local/lib/node_modules/n8n/node_modules/axios/lib/core/settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (/usr/local/lib/node_modules/n8n/node_modules/axios/lib/adapters/http.js:269:11)\n    at IncomingMessage.emit (node:events:525:35)\n    at endReadableNT (node:internal/streams/readable:1358:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)"
    },
    "timestamp": 1661627552236,
    "message": "The service failed to process your request",
    "node": {
        "parameters": {
            "authentication": "accessToken",
            "resource": "file",
            "operation": "share",
            "path": "/banana-whole-1.gif",
            "shareType": 3,
            "options": {}
        },
        "id": "504b6ebb-3f70-47be-8593-a6641cd19c4e",
        "name": "Nextcloud",
        "type": "n8n-nodes-base.nextCloud",
        "typeVersion": 1,
        "position": [
            460,
            300
        ],
        "credentials": {
            "nextCloudApi": {
                "id": "4",
                "name": "Dragan's filestore account"
            }
        }
    },
    "httpCode": "501",
    "description": null
}
```

</details>

**Comment by Joffcom** [comment]
Hey @despens,

Can you try 0.192.0? It looks like we may have already fixed this issue, I have just tested on 0.193.2 and sharing appears to be working.

**Comment by despens** [comment]
Thanks @Joffcom I just deployed the latest version and indeed sharing works!

There is another issue with the sharing functionality, but I'll make a separate issue for that.

Thanks a ton!
